rocAL/rocAL/include/meta_data/meta_data.h

```diff
@@ -25,6 +25,7 @@ THE SOFTWARE.
 #include <utility>
 #include <vector>
 #include <memory>
+#include <cstring>
```

Comment:
Where is this new include used?

Reply:
memcpy is used here hence this is included